mizar/dp/mizar/operators/endpoints/endpoints_operator.py

```diff
@@ -109,15 +110,34 @@ def create_scaled_endpoint(self, name, spec):
         ep.set_mac(self.rand_mac())
         ep.set_type(OBJ_DEFAULTS.ep_type_scaled)
         ep.set_status(OBJ_STATUS.ep_status_init)
+        backends = set()
+        response = kube_get_endpoints(self.core_api, name, namespace)
+        if response and response.subsets:
+            backends.add(response.subsets[0].addresses[0].ip)
+            ep.set_backends(list(backends))
+        logger.info(
+            "Added backends: {} to scaled endpoint {} ".format(backends, name))
         ep.create_obj()
-        self.annotate_builtin_endpoints(name)
+        self.annotate_builtin_endpoints(name, namespace)
```

Comment:
I kept the annotation filters here because system endpoints such as the Kube-controller-manager, and Kube-scheduler get updated continuously (about every few seconds).